### PR TITLE
Upgrade konnectivity to v0.0.15

### PR DESCRIPTION
**Pull request by mvladev**
**How to categorize this PR?**

/area networking
/kind TODO
/priority cleanup

**What this PR does / why we need it**:

The new `0.0.15` version fixes several issues (e.g memory leaks). Keep alive time is also reduced to 1min to make sure to keep connections alive for agents.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
Konnectivity components are upgraded to `v0.0.15`
```

### CODE REVIEW

**Review by rfranzke — DISMISSED** [review]
/lgtm

File: charts/images.yaml [hunk 1]
```diff
@@ -89,7 +89,7 @@ images:
 - name: konnectivity-server
   sourceRepository: github.com/gardener/replica-reloader
   repository: eu.gcr.io/gardener-project/gardener/replica-reloader
-  tag: "v0.1.0"
+  tag: "v0.2.0-konnectivity-server-v0.0.15"
```
Comment (vpnachev) [inline charts/images.yaml:92]: I do not see tag/release/branch related to this version here https://github.com/gardener/replica-reloader. Why?

Comment (vpnachev) [reply charts/images.yaml:92]: Ok, it is mentioned here https://github.com/gardener/replica-reloader#docker-images.

**Review by vpnachev — DISMISSED** [review]
/lgtm

**Comment by rfranzke** [comment]
Can you check the failing test?

**Comment by mvladev** [comment]
updated the PR.

**Review by vpnachev — APPROVED** [review]
/lgtm

**Pull request merged**